$OpenBSD: patch-src_mumble_AudioInput_cpp,v 1.1 2021/12/15 22:25:39 kn Exp $

a8e0797f5 "FIX(client): Update rnnoise-src submodule";  remove with next update

Index: src/mumble/AudioInput.cpp
--- src/mumble/AudioInput.cpp.orig
+++ src/mumble/AudioInput.cpp
@@ -106,7 +106,7 @@ AudioInput::AudioInput() : opusBuffer(g.s.iFramesPerPa
 #endif
 
 #ifdef USE_RNNOISE
-	denoiseState = rnnoise_create();
+	denoiseState = rnnoise_create(nullptr);
 #endif
 
 	qWarning("AudioInput: %d bits/s, %d hz, %d sample", iAudioQuality, iSampleRate, iFrameSize);
